src/main/java/uk/gov/hmcts/reform/em/hrs/domain/HearingRecording.java

```diff
@@ -58,6 +59,8 @@ public class HearingRecording {
 
     private boolean deleted;
 
+    @Value("$ttlSet:false")
```

Comment:
what is this?

Reply:
It was an initial thought to set the boolean value to false, but the same is accomplished from the db update. Removing it